server/punishments.ts

```diff
@@ -424,26 +380,20 @@ const Punishments = new (class {
 			buf += `${ip}\tSHARED\t${note}\r\n`;
 		});
 
-		FS(SHAREDIPS_FILE).write(buf);
+		return FS(SHAREDIPS_FILE).write(buf);
 	}
 
 	/*********************************************************
 	 * Adding and removing
 	 *********************************************************/
 
-	/**
-	 * @param {User | string} user
-	 * @param {Punishment} punishment
-	 * @return {User[]}
-	 */
-	punish(user, punishment) {
+	punish(user: User | ID, punishment: Punishment) {
 		if (typeof user === 'string') return Punishments.punishName(user, punishment);
 
-		/** @type {Set<string>} */
-		const userids = new Set();
-		/** @type {Set<string>} */
-		const ips = new Set();
-		const affected = user.getAltUsers(PUNISH_TRUSTED, true);
+		const userids: Set<string> = new Set();
```

Comment:
Same thing for set declarations.
```suggestion
		const userids = new Set<string>();
```